### DISCUSSION TITLE
Custom .cirrus.yml  file location

### DISCUSSION THREAD

**Discussion by barcode** [cirruslabs/cirrus-ci-docs#1131] [Q&A] [answered]
Hi,
is it possible to use a custom location for the config file (e.g. `ci_files/cirrus/.cirrus.yml`) or does it always have to be in the root dir of a repo?

If it is possible: How can i configure this on https://cirrus-ci.com ?

**Comment by fkorotkov** [comment, marked as answer]
It's not possible to use another location. It should be either `.cirrus.yml` or `.cirrus.star` in the root of your repository. If you'd like to split the YAML config file, as an option you can try to use Starlark:

```python
# .cirrus.star in the root of the repository
load("cirrus", "fs")

def main(ctx):
  return fs.read("ci_files/cirrus/.cirrus.yml")
```
